spec/Dockerfile.erb

```diff
@@ -2,9 +2,6 @@ FROM <%= reg_username %>cantaloupe_<%= cantaloupe_version %>
 
 USER root
 
-# this is only for testing purposes, it is not included in the dockerhub images
-ENV DELEGATE_URL = "https://raw.githubusercontent.com/UCLALibrary/cantaloupe-delegate/master/lib/delegates.rb"
```

Comment:
Rather than hard coding here we could pass in the travis ENV matrix and include it if it's set.

Reply:
Are you sure you're looking at the correct side of the diff? I agree, I deleted this.

Reply:
You're right. I was seeing that backwards.